test/jdk/javax/swing/plaf/basic/BasicMenuUI/4983388/bug4983388.java

```diff
@@ -56,21 +69,21 @@ private static void createAndShowGUI() {
         JMenu menu = new JMenu("File");
         menu.setMnemonic('F');
         menuBar.add(menu);
-        frame = new JFrame();
+        menu.addMenuListener(new TestMenuListener());
+
+        frame = new JFrame("bug4983388");
         frame.setJMenuBar(menuBar);
         frame.setLocationRelativeTo(null);
-        frame.pack();
+        frame.setSize(250, 100);
         frame.setVisible(true);
-        MenuListener listener = new TestMenuListener();
-        menu.addMenuListener(listener);
     }
 
     public static void main(String[] args) throws Exception {
-
         try {
             UIManager.setLookAndFeel("com.sun.java.swing.plaf.gtk.GTKLookAndFeel");
         } catch (UnsupportedLookAndFeelException | ClassNotFoundException ex) {
-            System.err.println("GTKLookAndFeel is not supported on this platform. Using defailt LaF for this platform.");
+            System.err.println("GTKLookAndFeel is not supported on this platform. "
+                               + "Using default LaF for this platform.");
         }
 
         SwingUtilities.invokeAndWait(new Runnable() {
```

Comment:
`createAndShowGUI` method is `static`, I suggest using method reference.

Reply:
Method references can be used in non-static context by providing an instance to the left of the double colon.

I did it on Friday but I forgot to push the change.